docs/.vitepress/config.js

```diff
@@ -287,4 +290,21 @@ export default defineConfig({
       },
     },
   },
+  vite: {
+    plugins: [
+      uno({
+        presets: [
+          presetWarp({ usePixels: true, usePreflight: true }),
+          presetDocs(),
+        ],
+        shortcuts: [
+          {
+            'ex-font': 'text-sm font-bold font-mono text-white',
+            'ex-box':
+              'ex-font p-24 rounded-4 shadow-xl flex items-center justify-center',
+          },
+        ],    
+      }),
+    ],
+  },
```

Comment:
this is just a refactoring as I figured we do not need two extra files for the config when we can have all in one.